### ISSUE TITLE
Angular 10 "scripts" chunk doesn't receive SRI

### ISSUE THREAD

**Issue by argelj289** [waysact/webpack-subresource-integrity#141]
I got a warning when executing ng build --subresource-integrity

```
chunk {common} common.js, common.js.map (common) 3.42 kB  [rendered]
chunk {components-digital-badge-management-digital-badge-management-module} components-digital-badge-management-digital-badge-management-module.js, components-digital-badge-management-digital-badge-management-module.js.map (components-digital-badge-management-digital-badge-management-module) 205 kB  [rendered]
chunk {components-events-events-module} components-events-events-module.js, components-events-events-module.js.map (components-events-events-module) 670 kB  [rendered]
chunk {components-locations-locations-module} components-locations-locations-module.js, components-locations-locations-module.js.map (components-locations-locations-module) 111 kB  [rendered]
chunk {components-message-management-message-module} components-message-management-message-module.js, components-message-management-message-module.js.map (components-message-management-message-module) 981 kB  [rendered]
chunk {components-people-people-module} components-people-people-module.js, components-people-people-module.js.map (components-people-people-module) 211 kB  [rendered]
chunk {components-power-bi-reports-power-bi-reports-module} components-power-bi-reports-power-bi-reports-module.js, components-power-bi-reports-power-bi-reports-module.js.map (components-power-bi-reports-power-bi-reports-module) 63.4 kB  [rendered]
chunk {components-rate-my-space-rate-my-space-module} components-rate-my-space-rate-my-space-module.js, components-rate-my-space-rate-my-space-module.js.map (components-rate-my-space-rate-my-space-module) 85.5 kB  [rendered]
chunk {components-request-local-support-request-local-support-module} components-request-local-support-request-local-support-module.js, components-request-local-support-request-local-support-module.js.map (components-request-local-support-request-local-support-module) 86 kB  [rendered]
chunk {components-useful-services-useful-services-module} components-useful-services-useful-services-module.js, components-useful-services-useful-services-module.js.map (components-useful-services-useful-services-module) 407 kB  [rendered]
chunk {content-layout-content-layout-module} content-layout-content-layout-module.js, content-layout-content-layout-module.js.map (content-layout-content-layout-module) 86.3 kB  [rendered]
chunk {default~components-digital-badge-management-digital-badge-management-module~components-events-events~3e96e4de} default~components-digital-badge-management-digital-badge-management-module~components-events-events~3e96e4de.js, default~components-digital-badge-management-digital-badge-management-module~components-events-events~3e96e4de.js.map (default~components-digital-badge-management-digital-badge-management-module~components-events-events~3e96e4de) 353 kB  [rendered]
chunk {main} main.js, main.js.map (main) 702 kB [initial] [rendered]
chunk {offices-offices-module} offices-offices-module.js, offices-offices-module.js.map (offices-offices-module) 548 kB  [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 321 kB [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 12.1 kB [entry] [rendered]
chunk {scripts} scripts.js, scripts.js.map (scripts) 663 kB [entry] [rendered]
chunk {styles} styles.js, styles.js.map (styles) 906 kB [initial] [rendered]
chunk {vendor} vendor.js, vendor.js.map (vendor) 7.05 MB [initial] [rendered]

WARNING in webpack-subresource-integrity: Cannot determine asset for chunk scripts, computed="undefined", available=scripts.js. Please report this full error message along with your Webpack configuration at https://github.com/waysact/webpack-subresource-integrity/issues/new
```

angular cli version: 10.1.3

webpack-subresource-integrity version: 1.4.1

webpack.config.js

![image](https://user-images.githubusercontent.com/40298377/103608199-befd0700-4f55-11eb-8957-3948651972dd.png)

**Comment by jscheid** [comment]
Which Webpack version are you using?

**Comment by argelj289** [comment]
webpack version is 4.44.1

**Comment by jscheid** [comment]
I think this is the same as discussed here: https://github.com/waysact/webpack-subresource-integrity/issues/133#issuecomment-720874670

**Comment by jscheid** [comment]
As far as I know this has been fixed in https://github.com/angular/angular-cli/issues/19829. Can somebody confirm?

**Comment by jscheid** [comment]
To my knowledge the fix for this was released in [angular-cli v11.2.0](https://github.com/angular/angular-cli/releases/tag/v11.2.0-next.0), I'm assuming that resolves the issue.
